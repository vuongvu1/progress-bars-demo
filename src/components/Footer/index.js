import React from 'react';
import styled from 'styled-components';

// #region Styles
const Wrapper = styled.footer`
  background-color: #20232a;
  color: #ffffff;
  width: 100%;
  padding: 0 20px;
  height: 50px;
  display: flex;
  justify-content: center;
  align-items: center;
`;
// #endregion Styles

const Footer = () => (
  <Wrapper>This is Footer</Wrapper>
);

export default Footer;
