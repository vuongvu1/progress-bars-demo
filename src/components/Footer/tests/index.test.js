import React from 'react';
import { render } from '@testing-library/react';

import Footer from '../index';

const renderComponent = (props = {}) => render(<Footer {...props} />);

describe('<Footer />', () => {
  it('should render <Footer />', () => {
    const { container } = renderComponent();
    expect(container).toMatchSnapshot();
  });
});
