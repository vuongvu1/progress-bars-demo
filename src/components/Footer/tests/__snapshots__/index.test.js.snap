// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render <Footer /> 1`] = `
<div>
  <footer
    class="sc-bdVaJa SyRAu"
  >
    This is Footer
  </footer>
</div>
`;
